src/main/java/com/faforever/client/patch/GameUpdaterImpl.java

```diff
@@ -69,7 +69,7 @@ public CompletableFuture<Void> update(FeaturedModBean featuredMod, Integer versi
     if (!NAMES_OF_FEATURED_BASE_MODS.contains(featuredMod.getTechnicalName())) {
       featuredModUpdateFuture = simModsUpdateFuture.thenCompose(aVoid -> modService.getFeaturedMod(FAF.getTechnicalName()))
           .thenCompose(baseMod -> updateFeaturedMod(baseMod, null))
-          .thenCompose(patchResult -> updateGameBinaries(patchResult.getVersion()))
+          .thenCompose(patchResult -> updateGameBinaries(new ComparableVersion(version.toString())))
```

Comment:
This won't work because this is just the binary. The lua files are still the wrong version as indicated by the updateFeaturedMod above. Really the version should go where that null is but I forget if there is a reason why it is currently null

Reply:
Oh now I remember why it is because allegedly the version corresponds to the version of the featured mod and not necessarily the base game. At least at the time this was written. So there wasn't really any info on the base game version so just always set it to latest. But that could have been misinterpreted and we can use it for the base game version

Reply:
But i think now this version number always corresponds to the base faf version but I don't know where the featuredmod version should be gotten from anymore. There is a mapping of numbers to version number in the replay header but I am unable to find the mapping itself to which featuredmod those correspond to

Reply:
Never mind I realized those correspond to the individual files. I will make a new pr to fix it all